src/sort/sort.less

```diff
@@ -1,4 +1,5 @@
 .sort-pf {
+  display: inline-block;
```

Comment:
I fear this change. sort-pf is used in many places by many applications in and out of toolbars. Why is this necessary?